app/uk/gov/hmrc/plasticpackagingtax/registration/views/models/BackButton.scala

```diff
@@ -18,4 +18,14 @@ package uk.gov.hmrc.plasticpackagingtax.registration.views.models
 
 import play.api.mvc.Call
 
-case class BackButton(title: String, call: Call, hiddenText: String)
+sealed trait BackButtonLink {
+  val title: String
+  val hiddenText: String
+  val call: Call
+}
+
+case class BackButton(override val title: String, override val call: Call, override val hiddenText: String) extends BackButtonLink
```

Comment:
I have added this for the javascripts as I did not want to touch the other back link as they work as it stand. Later if we decide to spend some time (which I am not sure if it is worth it) to do all the back link we can refactor this to use only one